utils.js

```diff
@@ -162,21 +167,64 @@ module.exports = {
   bluepill: 'https://i.imgur.com/mpiVwDX.jpg',
   msys: 'https://msys.qmk.fm/',
   coc: 'https://qmk.fm/coc/',
-  kbdfans: 'Against our best advice, KBDfans have chosen a less than ideal bootloader for their newer board revisions. You may have luck using <https://tinyurl.com/KBDFansFlashNew> for reference, but if you are still stuck you will have to visit KBDfans discord for support: <https://discord.gg/kbdfans>',
+  kbdfans: 'Against our best advice, KBDfans have chosen a less than ideal bootloader for their newer board revisions. The latest QMK Toolbox does support it which can be downloaded from <https://github.com/qmk/qmk_toolbox/releases>. Otherwise you may have luck using <https://tinyurl.com/KBDFansFlashNew> for reference, but if you are still stuck you will have to visit KBDfans discord for support: <https://discord.gg/kbdfans>',
+  sonixinvite: 'https://discord.gg/q8VjhRgzRw',
+  sonix: 'Support is maintained in another fork and is not part of QMK yet.\n\nGitHub: <https://github.com/SonixQMK/>\nDiscord: ',
+  openrgbinvite: 'https://discord.com/invite/AQwjJPY',
+  openrgb: 'Support is maintained in another fork and is not part of QMK yet.\n\nSite: <https://openrgb.org/>\nGitHub: <https://github.com/Kasper24/QMK-OpenRGB>\nDiscord: ',
+  vialinvite: 'https://discord.gg/6Ybrtvj6Ae',
+  vial: 'Support is maintained in another fork and is not part of QMK yet.\n\nSite: <https://get.vial.today/>\nGitHub: <https://github.com/vial-kb/vial-qmk>\nDiscord: ',
   vid: 'https://yanfali.github.io/qmk_usb_usage/',
   vidq: 'https://www.the-sz.com/products/usbid/index.php',
   lighting: '**Backlight:** Single color, per key, not individually addressable (all on or off)\n**RGBLight:** Underglow RGB, linear design (single chain), individually addressable\n**LED Matrix:** Single color, per key, individually addressable\n**RGB Matrix:** RGB, per key, individually addressable\n**Indicators:** Scroll/Caps/Num Lock LEDs',
-
+  markdown: 'Inserting triple backticks at the beginning and end of your code will create a code block. You can specify a language after the first triple back tick, applicable options would be c, make, python and markdown.\n \n\\\`\\\`\\\`c\n\`\`\`c\nif (is_fancy_example) {\n   return false;\n}\n\`\`\`\n\\\`\\\`\\\`',
+  inputlanguage: 'Keyboards don\'t send characters, it uses keycodes. The input language set by the OS maps those keycodes to characters. If you have swapped round the keys in your keyboard firmware try setting your OS input language to English, otherwise make sure you have the correct input language selected in your OS.\n\nThere\'s also keymap header files that map specific languages to keycodes. Follow this link for more informaiton <https://docs.qmk.fm/#/reference_keymap_extras?id=language-specific-keycodes>',
```

Comment:
Tiny bit of wordsmithing, because I'm an occasionally nosy collaborator. :stuck_out_tongue: 

```suggestion
  inputlanguage: 'Keyboards don\'t send characters, they send keycodes. The input language set by the OS maps those keycodes to characters. If you have swapped round the keys in your keyboard firmware try setting your OS input language to English, otherwise make sure you have the correct input language selected in your OS.\n\nThere are also keymap header files that map specific languages to keycodes. Follow this link for more information <https://docs.qmk.fm/#/reference_keymap_extras?id=language-specific-keycodes>',
```